elements/pfe-progress-steps/src/pfe-progress-steps.js

```diff
@@ -53,14 +53,19 @@ class PfeProgressSteps extends PFElement {
 
   constructor() {
     super(PfeProgressSteps, { type: PfeProgressSteps.PfeType });
+    this._resizeObserver = new ResizeObserver(this._build.bind(this));
   }
 
   connectedCallback() {
     super.connectedCallback();
-    this._build();
+    // this will call _build initially and estabilish a resize observer for this element
+    this._resizeObserver.observe(this);
```

Comment:
Can we add an IE11 check before this?

Reply:
checking for presence of resizeObserver first.  If not, just build it one time.  If it's IE11, then the _build command will catch it.